test/testgroups/emptyresourcetests.go

```diff
@@ -0,0 +1,14 @@
+package testgroups
+
+import (
+	"testing"
+
+	"github.com/operator-framework/operator-marketplace/test/testsuites"
+)
+
+// EmptyResourceTestGroup runs tests without creating any resources upfront. Use this group if you can't
+// rely on the framework to clean up after your tests.
+func EmptyResourceTestGroup(t *testing.T) {
```

Comment:
This is similar to the test group that I'm creating in my PR, since my tests didn't require any prior resources. Should we use the same test group in this case?
Another question: should test groups be named for the resources they create (or don't create), or should they be used to organize similar/ related tests?

Reply:
Honestly, there's no reason to create a test group if the test suites don't rely on any resources. The only benefit seems to be that the `TestMarketplace` function remains clean.

If we do want to go down the path of creating a test group for tests that don't require any setup, could we change the name? How does `EmptyResource` convey that no resources were created? What is an `EmptyResource`? Could we go with something clearer, such as `NoResourceSetupTestGroup`? Or just `NoSetupTestGroup`

Reply:
I have not looked into the test framework in details yet, are the following assumptions correct?
* A group of test(s) share the same setup and teardown
* The test(s) inside a group run sequentially, while different group(s) may run in parallel
 
I am trying to see what criteria to consider when we create test group(s).